test/distributed/test_dynamo_distributed.py

```diff
@@ -1176,6 +1180,42 @@ def f(x):
             for r in res[1:]:
                 self.assertEqual(res[0], r)
 
+    @unittest.skipIf(not HAS_GPU, "Inductor+gpu needs triton and recent GPU arch")
+    @patch.object(torch._dynamo.config, "enable_compiler_collectives", True)
+    @patch.object(torch._inductor.config, "max_autotune_gemm", True)
```

Comment:
as discussed you can use 

@patch.object(torch._inductor.config, "max_autotune_gemm_backends", "TRITON")

to ensure we always pick a triton template for this test